test/e2e/replica/replica_test.go

```diff
@@ -0,0 +1,82 @@
+package replica_test
```

Comment:
Please mention this additional e2e test in the description.

Reply:
Done. I've also added an additional basic README.